pkgs/tools/networking/fastd/default.nix

```diff
@@ -12,8 +23,27 @@ stdenv.mkDerivation rec {
     sha256 = "1p4k50dk8byrghbr0fwmgwps8df6rlkgcd603r14i71m5g27z5gw";
   };
 
-  nativeBuildInputs = [ pkg-config bison meson ninja ];
-  buildInputs = [ libuecc libsodium libcap json_c openssl ];
+  nativeBuildInputs = [
+    bison
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    json_c
+    libcap
+    libsodium
+    libuecc
+    openssl
+  ];
+
+  # some options are only available on x86
+  mesonFlags = lib.optionals (!stdenv.isx86_64 && !stdenv.isi686) [
```

Comment:
```suggestion
  mesonFlags = lib.optionals (!stdenv.isx86) [
```

Reply:
> error: attribute 'isx86' missing, at /home/hexa/git/nixos/master/pkgs/tools/networking/fastd/default.nix:42:32

Apparently not a thing.

Reply:
I am pretty sure I coped that from somewhere.